tests/store/model_registry/test_sqlalchemy_store.py

```diff
@@ -870,6 +870,12 @@ def search_versions(filter_string):
             set([1, 2, 3]),
         )
 
+        # search IN operator is case sensitive
+        self.assertEqual(
+            set(search_versions(f"run_id IN ('{run_id_1.upper()}','{run_id_2}')")),
+            set([2, 3]),
+        )
+
```

Comment:
Do we have tests for `run_id LIKE` and `run_id ILIKE`?

Reply:
+1